newdle/client/src/components/creation/CreationSuccessPage.js

```diff
@@ -38,7 +38,17 @@ export default function CreationSuccessPage() {
             invite:
           </p>
         )}
-        <div className={styles['newdle-link']}>{newdle.url}</div>
+        <Input
+          className={styles['newdle-link']}
+          disabled
+          action={{
+            icon: 'copy',
+            onClick: async () => {
+              await navigator.clipboard.writeText(newdle.url);
```

Comment:
You don't need to `await` here (and thus no `async` needed in the line above) since there's nothing you do after copying the link anyway.

Could be nice to show some kind of "copied" message though - but not sure if it's really needed (feel free to give it a try though), or use a CSS animation to style the button a bit differently to make it clear that something happened when you clicked it.